src/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -95,11 +95,35 @@
  * @author mccowan@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = IlluminaBasecallsToSam.USAGE,
-        usageShort = IlluminaBasecallsToSam.USAGE,
+        usage = IlluminaBasecallsToSam.USAGE_SUMMARY + IlluminaBasecallsToSam.USAGE_DETAILS,
+        usageShort = IlluminaBasecallsToSam.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class IlluminaBasecallsToSam extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Transforms a lane of Illumina data file formats (*.bcl, *.locs, *.clocs, *.qseqs, etc.) into the" +
+            "SAM or BAM file format."  ;
+    static final String USAGE_DETAILS = "<p>The IlluminaBasecallsToSam tool enables the transformation of Illumina binary call (*.bcl) data to unmapped SAM/BAM files.  " +
```

Comment:
I don't like the word "enable" in this context because it sounds like an accessory piece that is part of a process, whereas in fact it's the thing that does all the work.